box/orchestrator.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+./livepeer -orchestrator -aiWorker -aiModels ./box/aiModels.json -serviceAddr localhost:8935 -transcoder -v 6 -liveAITrickleHostForRunner 'host.docker.internal:8935' -monitor
```

Comment:
We need to change this `liveAITrickleHostForRunner` to `172.17.0.1` when running under Linux